__tests__/utils.test.js

```diff
@@ -304,3 +304,63 @@ describe("Get /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("Should insert the new comment into the database and return the updated information", () => {
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send({
+        username: "butter_bridge",
+        body: "Have you ever seen a Lavender corn snake?",
+      })
+      .expect(201)
+      .then((response) => {
+        const data = response.body.returnedComment[0];
+        expect(data.author).toEqual("butter_bridge");
+        expect(data.article_id).toEqual(2);
+        expect(data.body).toEqual("Have you ever seen a Lavender corn snake?");
+      });
+  });
+  test("Should return a 400 bad request if given an article_id that does not exist", () => {
```

Comment:
This wouldn't be a bad request if the article id is valid but doesn't exist, it would be a 404 for something missing.